portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SegmentsCreation.testcase

```diff
@@ -473,8 +477,12 @@ definition {
 	}
 
 	@description = "Feature ID: Legacy | Automation ID: LRAC-8480 | Test Summary: Create a Dynamic Individuals Segment with a criteria that uses is known and is not"
+	@ignore = "true"
 	@priority = "4"
 	test CanCreateDynamicSegmentWithIndividualCriteriaUsesIsKnownAndIsNot {
+
+		// CSV data source tests are no longer possible to run due to this ticket LRAC-11651
+
```

Comment:
Hey @manuferreira, I think this validation can be done using other criteria.

I think you can undo this change and create a ticket for us to refactor this test or you can do it yourself in your ticket